src/Core/src/Handlers/Application/ApplicationHandler.cs

```diff
@@ -83,5 +86,9 @@ protected override PlatformView CreatePlatformElement() =>
 		/// <param name="application">The associated <see cref="IApplication"/> instance.</param>
 		/// <param name="args">The associated command arguments.</param>
 		public static partial void MapCloseWindow(ApplicationHandler handler, IApplication application, object? args);
+
+#if ANDROID || IOS
```

Comment:
@MartyIX Glad you're here :) Could you maybe have a look at how Windows behaves?

Reply:
Does the theme not flow to the children? So it is not possible to set the theme on the window or the root content and get magic to flow?

I know in Maui itself, we use the dynamic resource system to push the theme to each child. But, I suppose if we had to we can maybe do the same on windows.

We already have a hack to toggle the theme on and off just to get dynamic colors to update. If there is a perf issue, we can push back and say the WinUI team has to either fix or provide some other way to set the theme like the rest of the development world.

Reply:
OK, maybe for another PR we try to see if the theme flows, and if not, we see how long it really takes to update all the children's themes. Maybe it is super fast? I think that when we "refresh the theme resources" with our hack that "sets the theme to unspecified and then back" for things like button backgrounds, it is so fast that the UI did not have time to notice the change. I think this is because it is sort of setting a flag for the next UI cycle, so if we update all the controls, it may not actually trigger a direct refresh but wait for the next UI "cycle" or "frame".